### PR TITLE
refactor: remove type change on click

### PR DESCRIPTION
**Pull request by matt-user**
closes FE-27

### CODE REVIEW

**Review by luizstacio — APPROVED** [review]

**Review by luizstacio — CHANGES_REQUESTED** [review]
We should add to this links a hover with color primary

**Comment by luizstacio** [comment]
Can you also add the hover on the block?
Screenshot 2023-11-13 at 2 09 56 PM

**Comment by luizstacio** [comment]
- When we are rendering a asset_id we don't need to have a link;
Screenshot 2023-11-13 at 2 10 25 PM

**Comment by luizstacio** [comment]
> Can you also add the hover on the block? Screenshot 2023-11-13 at 2 09 56 PM

Can you do this also?

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
can we make the address always link style, instead of only when hovering?

on this way the user knows it's a link, otherwise user needs to hover to know it's a link

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
image

add pointer to copy button

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
- add link style also when not hovering
- remove useless icon

image

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
before:
image

----
after
image

---
address size is looking way smaller than before, and also disaligned

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
image

not showing sender / recipient anymore

**Review by luizstacio — CHANGES_REQUESTED** [review]
The size and font weight should be the same of previous versions. It looks a bit bigger and bold now.
Screenshot 2023-11-15 at 8 12 32 PM
Screenshot 2023-11-15 at 8 12 19 PM

**Review by luizstacio — DISMISSED** [review]

File: packages/ui/src/components/Copyable/Copyable.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ export const Copyable = createComponent<CopyableProps, 'span'>({
         {children}
         <Tooltip content={tooltipMessage}>
           <IconButton
+            className="cursor-pointer"
```
Comment (LuizAsFight) [inline packages/ui/src/components/Copyable/Copyable.tsx:54]: this fix would be better on UI side ?

`<IconButton>` is a Button I'd expect it to be pointer by default, unless it's disabled/readOnly

Comment (LuizAsFight) [reply packages/ui/src/components/Copyable/Copyable.tsx:54]: this would also fix all other IconButtons

image

**Review by LuizAsFight — DISMISSED** [review]

**Review by luizstacio — APPROVED** [review]

**Pull request merged**